packages/codegen.go/src/example.ts

```diff
@@ -416,7 +417,7 @@ function generateFakeExample(goType: go.PossibleType, name?: string): go.Example
         return new go.NumberExample(goType.values[0].value as number, goType);
     }
   }
-  throw new Error(`do not support to fake example for none primitive type: ${goType}`);
+  throw new CodegenError('InternalError', `do not support to fake example for none primitive type: ${go.getTypeDeclaration(goType)}`);
```

Comment:
Why internal error?

Also the grammar of the sense is tough to parse. 
Should we say: "fakes do not support none primitive type: ........"

Reply:
Reworded the sentence.

For error cases where the codegen portion doesn't handle some case, I classified those as `InternalError` (compared to some case e.g. we expect `X` but got `Y`). We _could_ add a separate error code to separate the two however I don't think it would add a lot of value. In both cases we want to dump the stack and error message informing the user to open an issue. Whether or not it was because some precondition failed, or we just didn't handle some case is less important as the stack will tell us which case it was.